ISPP-12/SarandONGa
# FEAT(#436): CREATE AGE ALERT CHILDS

To test this pr, you need three things:
-Create a child with coming of age
-Change the line 111 in the script in main/template/layouts/master (to a time that you see reasonable and short, setting setInterval(verificarNinos(), smalltime))
-Allow browser notifications by clicking on the search area of ​​the browser as shown in the photo
![image](https://user-images.githubusercontent.com/80246365/229572546-15182dba-0b10-4ab4-bf52-0262f665d810.png)

The notifications will appear in the notification center of the operating system in which you are.
This is an additional measure to what the members of the VidesSur ong want, if what they want is a page to enter and show those children 18 years of age or older, that can be delegated to the task of filtering the list of children.
